lib/ansible/modules/cloud/azure/azure_rm_policysetdefinition.py

```diff
@@ -0,0 +1,370 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2019 Fan Qiu, (fanqiu@microsoft.com)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_policysetdefinition
+version_added: "2.9"
+short_description: Manage Azure Policy Set Definition
+description:
+    - Create, update and delet Azure policy set definition.
+
+options:
+    name:
+        description:
+            - Name of the policy set definition
+        required: true
+    management_group:
+        description:
+            - The ID of the management group.
```

Comment:
is it a  azure resource id? how to  get the management_group id?

Reply:
It isn't the azure resource id. This [link](https://docs.microsoft.com/en-us/azure/governance/management-groups/index) explain the management group. Perhaps, a new module for the management group is suitable?